### PR TITLE
Wifi Facade. OSX, Windows, Linux

### PR DESCRIPTION
**Pull request by kiok46**
This PR provides following features:

    - Get the available device's name and network information.
    - Scanning for available devices.
    - Connect to network.
    - Disconnect from network.

### CODE REVIEW

**Comment by kiok46** [comment]
Adding screenshots for OSX.

screen shot 2016-06-30 at 6 01 10 pm
screen shot 2016-06-30 at 6 01 22 pm

**Comment by kiok46** [comment]
Works smoothly in scanning for windows part, need to work a little on connecting with a network.

**Comment by kiok46** [comment]
could be merged.

File: plyer/facades/wifi.py [hunk 1]
```diff
@@ -0,0 +1,263 @@
+'''
+Wifi Facade.
+=============
+
+The :class:`Wifi` is to provide access to the wifi of your mobile/ desktop
+devices.
+It currently supports enabling, connecting, disconnecting, scanning and
+getting network information.
+
+Usage example
```
Comment (akshayaurora) [inline plyer/facades/wifi.py]: Usage example should be short and simple, like 
```
from player import wifi
```
To enable wifi just do:: 

```
wifi.enable()
```

**Comment by akshayaurora** [comment]
@kiok46 thanks!

**Pull request merged**